src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -23,8 +22,6 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
     <Compile Remove="CpuMathUtils.netstandard.cs" />
-    <!-- This is only needed until https://github.com/dotnet/corefx/issues/31064 is addressed. -->
-    <PackageReference Include="System.Runtime.Intrinsics.Experimental" Version="4.6.0-preview1-26708-04" />
```

Comment:
I guess you're deleting this because this is now inbox in 3.0? Or am I missing something.

Reply:
Correct. https://github.com/dotnet/corefx/issues/31064 has been fixed for some time.